packages/rancher-istio/charts/templates/istio-uninstall-job.yaml

```diff
@@ -39,7 +42,9 @@ spec:
             name: istio-installer-overlay
         {{ end }}
       serviceAccountName: istio-installer
+      {{- if .Values.global.rbac.pspEnabled }}
       securityContext:
         runAsUser: 101
         runAsGroup: 101
+      {{- end }}
```

Comment:
Why do we wrap this one in conditional but not the above one ?

Reply:
Originally the security context was only set when PSP was enabled, but now it needs to be on all the time - I have removed the job level security context and I am only keeping the container level securityContext so this is not longer an issue.